update_multiarch.sh

```diff
@@ -43,7 +45,11 @@ do
 			btypes=$(parse_vm_entry "${vm}" "${version}" "${package}" "${os}" "Type:")
 			dir=$(parse_vm_entry "${vm}" "${version}" "${package}" "${os}" "Directory:")
 
-			for build in ${builds}
+            if [ ! -z "${build_arg}" ]; then
```

Comment:
Looks like you have spaces instead of tabs here

Reply:
changed spaces to tabs